### DISCUSSION TITLE
Sidebar missing, has this been removed?

### DISCUSSION THREAD

**Discussion by elixd** [datenstrom/community#531] [Ask a question] [answered]
I have found on the web that some Yellow CMS sites have a side bar. For example here is a [page](https://in.10437.lima.zone/help/how-to-make-a-blog) where is written that it is sufficient to add a sidebar.md in shared folder.

But it does not work for me. Has this functionality been removed? Any plans to bring it back?

I am not a web developer. My goal is to have a blog with a sidebar where I can list all tags and categories.

**Comment by schulle4u** [comment, marked as answer]
The sidebar feature has been moved to themes. This means that the default layout files provide a one-column layout, but it is possible to adapt any theme for Yellow and create individual layout files for your purpose. 

BTW: The official and most up-to-date documentation can be found [here](https://datenstrom.se/yellow/help/).

HTH
Steffen
